dotnet/src/webdriver/PrintOptions.cs

```diff
@@ -50,12 +50,24 @@ public class PrintOptions
         private const double DefaultPageHeight = 21.59;
         private const double DefaultPageWidth = 27.94;
         private const double CentimetersPerInch = 2.54;
-
+        public static PageSize A4 { get; } = new PageSize { Width = 21.0, Height = 29.7 }; // cm
```

Comment:
`PageSize` is a mutable type. If a user does `PrintOptions.A4.Height = 0`, then it will be mutated for every future access.

For that reason, each property access should make a new `PageSize` instance (applies to all 4 presets).

```suggestion
        public static PageSize A4 => new PageSize { Width = 21.0, Height = 29.7 }; // cm
```